docker-compose.yml

```diff
@@ -56,7 +56,6 @@ services:
       --execution-endpoint=http://geth:8551
       --execution-jwt=/opt/jwt/jwt.hex
       --datadir=/opt/app/beacon/
-      --builder=${BUILDER_API_RELAY_URL:-https://0xaa58208899c6105603b74396734a6263cc7d947f444f396a90f7b7d3e65d102aec7e5e5291b27e08d02c50a050825c2f@holesky.titanrelay.xyz/}
```

Comment:
This value should be the mev boost address, like we have in our OG CDVN: https://github.com/ObolNetwork/charon-distributed-validator-node/blob/5737840a6073492a17d72b3c2ff76cd25cf28369/docker-compose.yml#L59

Reply:
Fixed!